src/transaction.js

```diff
@@ -200,18 +200,46 @@ Transaction.prototype.hashForSignature = function (inIndex, prevOutScript, hashT
   txTmp.ins.forEach(function (txIn) {
     txIn.script = Script.EMPTY
   })
-  txTmp.ins[inIndex].script = hashScript
+  txTmp.ins[inIndex].script = hashScript;
```

Comment:
Please remove the semi-colons throughout, they break our test suite as we use [standard](https://github.com/feross/standard) on this project.